src/main/common/utils/freeMediationUtil.ts

```diff
@@ -1,12 +1,57 @@
-import { YesNoOption } from 'claims/models/response/core/yesNoOption'
-import { FreeMediation } from 'forms/models/freeMediation'
+import { MediationDraft } from 'main/features/mediation/draft/mediationDraft'
+import { YesNoOption } from 'main/app/claims/models/response/core/yesNoOption'
+import { ResponseDraft } from 'main/features/response/draft/responseDraft'
+import { Claim } from 'main/app/claims/models/claim'
+import { FreeMediationOption } from 'main/app/forms/models/freeMediation'
+import { CompanyDetails } from 'main/app/forms/models/companyDetails'
 
 export class FreeMediationUtil {
-  static convertFreeMediation (freeMediation: FreeMediation): YesNoOption {
+
+  static getFreeMediation (mediationDraft: MediationDraft): YesNoOption {
+    const freeMediation = mediationDraft.youCanOnlyUseMediation
+
     if (!freeMediation || !freeMediation.option) {
       return YesNoOption.NO
     } else {
       return freeMediation.option as YesNoOption
     }
   }
+
+  static getMediationPhoneNumber (claim: Claim, mediationDraft: MediationDraft, draft?: ResponseDraft): string {
+    if (mediationDraft.canWeUseCompany) {
+      if (mediationDraft.canWeUseCompany.option === FreeMediationOption.YES) {
+        return mediationDraft.canWeUseCompany.mediationPhoneNumberConfirmation
+      } else {
+        return mediationDraft.canWeUseCompany.mediationPhoneNumber
+      }
+    } else if (mediationDraft.canWeUse) {
+      if (mediationDraft.canWeUse.option === FreeMediationOption.YES) {
+        if (!claim.isResponseSubmitted() && draft) {
+          return draft.defendantDetails.mobilePhone.number ? draft.defendantDetails.mobilePhone.number : undefined
+        } else {
+          return claim.claimData.claimant.mobilePhone ?
+            claim.claimData.claimant.mobilePhone : mediationDraft.canWeUse.mediationPhoneNumber
```

Comment:
Similarly, this can be simplified to `return claim.claimData.claimant.mobilePhone || mediationDraft.canWeUse.mediationPhoneNumber`

Reply:
done